tests/foreman/ui/test_settings.py

```diff
@@ -353,9 +353,46 @@ def test_positive_update_email_delivery_method_sendmail():
     :CaseImportance: Critical
 
     :CaseLevel: Acceptance
-
-    :CaseAutomation: notautomated
     """
+    mail_config_default_param = {
+        "delivery_method": "",
+        "email_reply_address": "",
+        "email_subject_prefix": "",
+        "sendmail_arguments": "",
+        "sendmail_location": "",
+        "send_welcome_email": "",
+    }
+    for content in mail_config_default_param:
+        mail_config_default_param[content] = [
+            ele.value
+            for ele in entities.Setting().search(query={'search': f'name={content}'})
+        ][0]
+
+    mail_config_new_params = {
+        "delivery_method": "Sendmail",
+        "email_reply_address": "root@{}".format(ssh.settings.server.hostname),
+        "email_subject_prefix": [gen_string('alpha')],
+        "sendmail_location": "/usr/sbin/sendmail",
+        "send_welcome_email": "Yes",
+    }
+
+    with session:
+        try:
+            for mail_content in mail_config_new_params:
+                session.settings.update(
+                    "{}".format(mail_content), "{}".format(mail_config_new_params[mail_content])
```

Comment:
Haven't noticed that before:
1. Useless use of format. `mail_content` is string.
2. Iterating over keys and looking up value on each iteration is not *wrong*, but isn't very pythonic, either. Better to use `.items()` on dict and iterating over key/value pair:
```python
for mail_content, mail_content_value in mail_config_new_params.items():
    session.settings.update(mail_content, mail_content_value)  # is value a string? If not, we might need this .format() anyway
```